apps/galaxy/src/layouts/HeaderLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import Head from "next/head";
+
+interface HeaderLayoutProps {
+  children: ReactNode;
+  title?: string;
+}
+
+// pass title value to set the head title， e.g. In 404 page, write <HeaderLayout title="404"></HeaderLayout>
+const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
```

Comment:
Try not to use `React.FC`, if you google this, there are lots of articles talking about this being deprecated.

Also, if I understood this correctly, the purpose of using `React.FC` is to provide type def for `children` property.